vue/components/organisms/personalization/personalization.vue

```diff
@@ -248,7 +248,20 @@ export const Personalization = {
 
         this.$bus.bind("initials_extra", initialsExtra => {
             if (this.diffInitialsExtra(initialsExtra, this.state.initialsExtra)) {
-                this.state.initialsExtra = initialsExtra;
+                const groups = Object.keys(initialsExtra);

```

Comment:
we need to generalize this logic as it appears in many places...

Reply:
For now, this is the first time it appears on white+commons-pluginus. Should this be moved to a `initialsEmpty(initialsExtra)` on ripe-sdk or just to [logic](https://github.com/ripe-tech/ripe-commons-pluginus/blob/master/vue/mixins/logic.js)? I think it could be in the SDK since the logic is there already and is useful enough to be exposed to clients.

Reply:
Isn't this https://github.com/ripe-tech/ripe-commons-pluginus/blob/master/vue/store.js#L98 pretty much the same logic?